### PR TITLE
WIP Added openfisca, truncated data

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?

WIP

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] tested in Chrome
- [ ] tested in Firefox
- [ ] tested in IE
- [ ] tested in Safari
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

File: src/components/FirstTimeApplicant/WizardFormFirstPage.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 import {Field, reduxForm} from 'redux-form';
+import {connect} from 'react-redux';
 import validate from '../../helpers/validate';
 import renderField from './renderField';
 import {scrollToFirstError} from '../../components/Forms/FormScroll';
-// import ReactAutocomplete from 'react-autocomplete';
-import jsonQuery from 'json-query';
-import RatesRebatesTable from './rates-rebates';
+import ReactAutocomplete from 'react-autocomplete';
+// import jsonQuery from 'json-query';
```
Comment (Br3nda) [inline src/components/FirstTimeApplicant/WizardFormFirstPage.js]: needs removal?

**Comment by dlouise64** [comment]
https://app.atomic.io/d/n3QobPHh1WXK?page=a5c804aa-5c29-42b8-b68d-d138eb4aa81c

**Comment by dlouise64** [comment]
Have closed #80 as this is the same page

**Comment by dlouise64** [comment]
branch feature/eligibility_page

**Comment by dlouise64** [comment]
@Br3nda Closing this as UI and method for Openfisca has changed a bit.

**Pull request closed without merging**